examples/05_transformers.ipynb

```diff
@@ -0,0 +1,849 @@
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# from os import sys\n",
+    "# sys.path.append(\"..\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Using sktime transformers\n",
+    "\n",
+    "Transformers are modularized data processing steps commonly used in machine learning. `sktime` introduces transformer types related to time series, which can be used in pipeline constructs across learning tasks. For instance, the same time series feature extractor could be used in a forecasting pipeline, or a time series classification pipeline.\n",
+    "\n",
+    "Note: the term \"transformer\" is used in the sense of `scikit-learn`. This should be distinguished from the \"transformer\" referring to specific neural network architecture.\n",
+    "\n",
+    "`sktime` provides common interfaces to different types of transformations for stand-alone or composition use. As all `sktime` estimators, transformers expose parameters which can be tuned inside pipelines and composites.\n",
+    "\n",
+    "**Section 1** provides an overview of the different kinds of transformers in `sktime`.\n",
+    "\n",
+    "**Section 2** discusses in more detail time series and panel transformers, and common uses in pipelines.\n",
+    "\n",
+    "**Section 3** discusses in more detail pairwise transformers, including time series distances and time series kernels.\n",
+    "\n",
+    "**Section 4** gives an introduction to how to write custom transformers compliant with the `sktime` interface."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Table of Contents\n",
+    "\n",
+    "* [1. Basic forecasting workflows](#chapter1)\n",
+    "    * [1.1 Data container format](#section_1_1)\n",
+    "    * [1.2 General transformer signature - time series transformers](#section_1_2)   \n",
+    "    * [1.3 General transformer signature - pairwise series transformers](#section_1_3)         \n",
+    "    * [1.4 General transformer signature - pairwise transformers](#section_1_4) \n",
+    "    * [1.5 Searching for transformers](#section_1_5)         \n",
+    "* [2. Time series transformers in detail](#chapter2)\n",
+    "* [3. Pairwise transformers in detail](#chapter3)  \n",
+    "* [4. Extension guide - implementing your own transformer](#chapter4)        \n",
+    "* [5. Summary](#chapter5)          "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### package imports"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Overview of transformers in `sktime` <a class=\"anchor\" id=\"chapter1\"></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This section explains the different types of transformers found in `sktime`.\n",
+    "\n",
+    "There are four main types of transformation in `sktime`:\n",
```

Comment:
Would it be possible to include the names or classes/mixins of the different transformers before the explanations in the list below?